test_requirements.txt

```diff
@@ -1,4 +1,5 @@
 future
 requests
+pytz
```

Comment:
we don't need to duplicate requirements, see https://github.com/html5lib/html5lib-python/blob/master/requirements-test.txt :smile: